deployments/helm/values.yaml

```diff
@@ -34,21 +70,30 @@ deployment:
     requests:
       cpu: 500m
       memory: 1Gi
+  ## Optionally specify extra environment variables to add to papyrus container
+  extraEnv: []
```

Comment:
the word `extra` in `extraEnv` implies that we have some default/pre-existing env vars we support. since we don't have such we don't need the `extra` part.